notebooks/getting_started/bq_dataframes_template.ipynb

```diff
@@ -0,0 +1,1848 @@
+{
```

Comment:
<p>Nit: Typo with leaarn. Should be learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1200/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Good catch, done!